### DISCUSSION TITLE
Wildcard SSL Cert with Namecheap

### DISCUSSION THREAD

**Discussion by ghostersk** [NginxProxyManager/nginx-proxy-manager#1370] [Q&A]
I saw there is no Namecheap provider in option when I try to create DNS challenge. Correct me if I am wrong and there is some other way how to get wildcard cert for domain with Namecheap automatically, but i read that it is not working.

I found this [link](https://simon.aldrich.cc/blog/2021/04/automating-lets-encrypt-wildcard-certificates-with-the-namecheap-api/) for python script what should create the certificate automatically. My question is how i could add those certificates when they are created to the Nginx PRoxy Docker, as I noticed I cannot install there things, and also Certificates are stored in some special way there.
If I am able to generate the certificate with the script, then I would just store it in some location where Nginx PM can find it.
Any suggestions?

**Comment by EnzoAlbornoz** [comment]
Point the NS records from your domain to Cloudflare. Then, use their DNS method to obtain the certificate. You will continue with you domain at namecheap  and you don't need any credit card to keep the DNS at Cloudflare
